lib/mix/tasks/show_vars.ex

```diff
@@ -2,31 +2,37 @@ defmodule Mix.Tasks.ShowVars do
   @shortdoc "Show the names of the environment variables to use"
 
   @moduledoc """
-  This taks will look at the EnvVar.Config you have defined
+  This taks will look at the config you have defined
   and return the list of environment variables you will need
   to supply to the release at runtime.
   """
 
   use Mix.Task
 
   def run([prefix]) do
-    Mix.Task.run("compile")
-
-    # Dynamic module name avoids compilation warnings.
-    config_module = EnvVar.Config
-
-    if Code.ensure_compiled?(config_module) do
-      config = config_module.config()
-      EnvVar.Provider.show_vars(prefix: prefix, env_map: config)
-    else
-      Mix.raise("""
-      You must include a module called EnvVar.Config and supply
-      a config/0 function that returns the release config.
-      """)
-    end
+    Mix.Project.config()
+    |> Keyword.fetch!(:releases)
+    |> fetch_only_release!()
+    |> Keyword.fetch!(:config_providers)
+    |> find_env_var_provider_config!()
+    |> Keyword.put(:prefix, prefix)
+    |> EnvVar.Provider.show_vars()
   end
 
   def run(_args) do
     Mix.raise("You must specify a single prefix, or '' if you want none")
   end
+
+  defp fetch_only_release!([{_name, config}]), do: config
+  defp fetch_only_release!(_other), do: Mix.raise("There are multiple releases defined")
```

Comment:
Should we/can we exit with a 1 here (just to make sure this works in case we use it in a script and rely on the exit value to detect that something went wrong).

Reply:
`Mix.raise/1` already exits the system with exit code 1.